src/README.md

```diff
@@ -49,3 +49,49 @@ const App = () => (
 It'll grab **Open Sans** from Google's CDN [through a CSS import](https://github.com/zopaUK/react-components/blob/master/src/components/styles/Fonts.tsx#L3-L5).
 
 You're free to use this technique or add the dependency manually to your base HTML `<head />`
+
+### Spacing
+
+Spacing is included as part of the GlobalStyles component and is split into margin and padding each can be used as atomic classes on your components.
+
+We follow the same guidelines set out in Tailwind
+
+Margin: https://tailwindcss.com/docs/margin/
+
+Padding: https://tailwindcss.com/docs/padding/
+
+The number you define in the class selector refers to the index of the spacing constants set out here:
+
+```ts static
+const spacing = {
+  '0': '0',
+  '1': '4px',
+  '2': '8px',
+  '3': '16px',
+  '4': '24px',
+  '5': '32px',
+  '6': '40px',
+  '7': '56px',
+  '8': '64px',
+  '9': '104px',
```

Comment:
with regards to my previous comment, having it as an object is one thing, I was referring more to the fact that if you decide to add a value somewhere in between, you'll have to bump the rest which. Any chance we can add a few more values at the beginning just to be sure that we don't have to go back and adjust it later down the road?

```
  '0': '0',
  '1': '4px',
  '2': '8px',
  '3': '12px',
  '4': '16px',
  '5': '20px',
  '6': '24px',
```

Reply:
these will be the ones used the most. It's much easier to add or throw in something at the other end of the range as they will be used less often